keybase/platform_darwin.go

```diff
@@ -95,6 +96,11 @@ func (c config) osVersion() string {
 	return strings.TrimSpace(result.Stdout.String())
 }
 
+// TODO: this only tells us the compile time arch
+func (c config) osArch() string {
+	return runtime.GOARCH
```

Comment:
This can be:
```go
cmd := exec.Command("uname", "-m")
var buf bytes.Buffer
cmd.Stdout = &buf
err := cmd.Run()
if err != nil {
    return runtime.GOARCH
}
return buf.String()
```